YouTube.js

```diff
@@ -166,22 +166,40 @@ var testCases = [
ves abundance has left us in want, our knowledge has made us cynical, our cleverness hard and unkind.We think too much and feel too littleMore than machinery we need humanity, more than cleverness we need kindness and gentleness; without these qualities, life will be violent and all will be lost.The aeroplane and the radio have brought us closer together, the very nature of these inventions cries out for the goodness in men, cries out for universal brotherhood for the unity of us all. Even now my voice is reaching millions throughout the world; millions of despairing men, women and little children, victims of a system that makes men torture and imprison innocent people.To those who can hear me I say \"Do not despair\".The misery that is now upon us is but the passing of greed, the bitterness of men who fear the way of human progress.The hate of men will pass and dictators die and the power they took from the people, will return to the people and so long as men die liberty will never perish.Soldiers! Don't give yourselves to brutes, men who despise you and enslave you, who regiment your lives, tell you what to do, what to think, what to feel!Who drill you, diet you, treat you like cattle, use you as cannon fodder!Don't give yourselves to these unnatural men!Machine men, with machine minds and machine hearts. You are not machines! You are not cattle! You are men!You have the love of humanity in your hearts, you don't hate, only the unloved hate. The unloved and the unnatural.Soldiers, don't fight for slavery, fight for liberty!In the seventeenth chapter of Saint Luke it is written: \"The kingdom of God is within man\". Not one man, nor a group of men, but in all men; in you!You, the people have the power, the power to create machines, the power to create happiness. You, the people have the power to make this life free and beautiful, to make this life a wonderful adventure, then in the name of democracy let's use that power, let us all unite! Let us fight for a new world, a decent world that will give men a chance to work, that will give you the future and old age and security. By the promise of these things, brutes have risen to power, but they lie. They do not fulfil their promise, they never will.Dictators free themselves but they enslave the people.Now let us fight to fulfill that promise, let us fight to free the world, to do away with national barriers, do away with greed, with hate and intolerance.Let us fight for a world of reason, a world where science and progress will lead to all men's happiness.Soldiers! In the name of democracy, let us all unite!The Great Dictator is a 1940 American comedy-drama film starring, written, produced, scored, and directed by Charlie Chaplin, following the tradition of many of his other films. Having been the only Hollywood filmmaker to continue to make silent films well into the period of sound films, this was Chaplin's first true talking picture as well as his most commercially successful film. (Wikipedia)",
```

Comment:
Not a big deal, but this may be closer to the original if we used `.innerHTML` and then `ZU.cleanTags` Now, the newlines are missing causing some sentences to start without any white space after a period. (may also require a pass through `ZU.unescapeHTML` afterwards if there are any HTML entities... and maybe some additional white space cleanup)